.pre-commit-config.yaml

```diff
@@ -35,6 +30,9 @@ repos:
         files: ^pandas
         exclude: ^pandas/tests
         args: [--select, "ANN001,ANN2", --fix-only, --exit-non-zero-on-fix]
+    -   id: ruff-format
+        # TODO: "." not needed in ruff 0.1.8
```

Comment:
If ruff 0.1.11 would cause a different formatting, it could be nice to also upgade ruff

Reply:
Yeah agreed. I would like to wait for this PR to go before the open pre-commit PR that updates ruff